dev/linearhooks/internal/handlers/handlers.go

```diff
@@ -141,8 +173,32 @@ func getTeamUUID(ctx context.Context, c graphql.Client, idOrKey string) (string,
 	return resp.Team.Id, nil
 }
 
-func moveIssueToTeam(ctx context.Context, c graphql.Client, issueId, teamId string) error {
-	_, err := lineargql.MoveIssueToTeam(ctx, c, issueId, teamId)
+func getProjectUUID(ctx context.Context, c graphql.Client, teamID, projectName string) (string, error) {
```

Comment:
```suggestion
func getProjectUUID(ctx context.Context, c graphql.Client, teamUUID, projectName string) (string, error) {
```

It's a bit confusing to use 'ID' here and 'UUID' at the caller.